prow/proxy-common.inc

```diff
@@ -56,7 +56,7 @@ if [[ -n "${GOOGLE_APPLICATION_CREDENTIALS:-}" ]]; then
   gcloud auth activate-service-account --key-file="${GOOGLE_APPLICATION_CREDENTIALS}"
 
   # Use RBE when logged in.
-  BAZEL_BUILD_RBE_INSTANCE="${BAZEL_BUILD_RBE_INSTANCE:-projects/istio-testing/instances/default_instance}"
+  BAZEL_BUILD_RBE_INSTANCE="${BAZEL_BUILD_RBE_INSTANCE-projects/istio-testing/instances/default_instance}"
```

Comment:
Accidental change?

Reply:
Intentional, I found we are still using RBE with centos which I think is wrong. The `:-` means it will set it even if `BAZEL_BUILD_RBE_INSTANCE=""` is set, `-` is needed to allow us to unset it

Reply:
Do you recall what was the issue here? While the the remote execution engine uses Ubuntu toolchain, I believe that CentOS should be fine using object cache.

Reply:
Let's continue discussion in https://github.com/istio/proxy/pull/3173.